libr/core/cconfig.c

```diff
@@ -3606,6 +3606,53 @@ R_API int r_core_config_init(RCore *core) {
 	return true;
 }
 
+R_API void radare2_rc(RCore *r) {
+	char* env_debug = r_sys_getenv ("R_DEBUG");
```

Comment:
swap those lines, better have the env-debug closer to the if

Reply:
we have other cases where we only use env vars as boolean, what about having r_sys_getenv_asbool() to avoid this heap massaging and custom checks here and there?